.goreleaser.yml

```diff
@@ -131,9 +168,6 @@ checksum:
 dockers:
   - goos: linux
     goarch: amd64
-    binaries:
```

Comment:
What's the plan for the docker images? This looks like we are leaving out both binaries, but I also couldn't find reference to the `binaries` syntax in the goreleaser docs (is it the same as `ids`?). What's the effect of these changes?

Reply:
`binaries` is deprecated & ignored in later versions (probably why it's not in the docs). I think `ids` is the replacement, but we don't need to specify it because goreleaser parses a non-specified list as short-hand for "include everything"